src/main/kotlin/telegram/Handling.kt

```diff
@@ -666,81 +646,96 @@ object MafiaHandler {
             parametrized(advertCommand) {
                 showAdMenu(ChatId.fromId(chatId), bot)
             }
+            parametrized(listActiveGamesCommand) {
+                showActiveGamesMenu(chatId, long(0), bot, int(1))
+            }
+            parametrized(listRecentGamesCommand) {
+                showRecentGamesMenu(chatId, long(0), bot, int(1))
+            }
             parametrized(updateCheckCommand) {
                 updateCheck(str(0))
                 showAdmin(chatId, long(1), bot)
             }
             parametrized(hostRequestCommand) {
-                showHostRequests(long(0), chatId, bot)
+                showHostRequests(long(0), chatId, bot, int(1))
             }
             parametrized(hostSettingsCommand) {
-                showHostSettings(long(0), chatId, bot)
+                showHostSettings(chatId, long(0), bot, int(1))
             }
             parametrized(adminSettingsCommand) {
-                val messageId = long(0)
-                showAdminListMenu(chatId, messageId, bot)
+                showAdminListMenu(chatId, long(0), bot, int(1))
             }
             parametrized(timeLimitOnCommand) {
-                val res = bot.sendMessage(ChatId.fromId(chatId), "Введите срок действия разрешения в днях:")
-                if (res.isSuccess) {
-                    val desc = res.get().messageId
-                    createAdminContext(desc, AdminState.HOST_TIME)
-                }
+                val msgId = sendMessage(
+                    bot,
+                    chatId,
+                    "Введите срок действия разрешения в днях:"
+                )
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard {
+                        createAdminContext(msgId, AdminState.HOST_TIME)
```

Comment:
Not a part of keyboard, this will probably result in deleting the whole keyboard of this message